conda/gateways/logging.py

```diff
@@ -141,8 +141,10 @@ def emit(self, record):
 
 @lru_cache(maxsize=None)
 def initialize_logging():
-    # root gets level ERROR; 'conda' gets level WARN and propagates to root.
+    # root gets level ERROR
+    # 'conda' gets level WARN and does not propagate to root.
     initialize_root_logger()
```

Comment:
IMO we should remove `initialize_root_logger`, AFAIK this is the source of logging issues in downstream projects (e.g., https://github.com/conda/conda-build/pull/5275#discussion_r1599124852).

Reply:
I have simply removed `initialize_root_logger`. I think this is justified in light of the fact that the whole logging hierarchy starts with an additional "root", `conda`, which is configured separately.